src/tests/api/BavHappyPath.test.ts

```diff
@@ -56,9 +58,31 @@ describe("Test BAV End Points", () => {
         const authCode = authResponse.data.authorizationCode;
         const authRepeatResponse = await authorizationGet(origSessionId);
         const authRepeatResponseCode = authRepeatResponse.data.authorizationCode;
-        expect(authCode).not.toEqual(authRepeatResponseCode); 
+        expect(authCode).not.toEqual(authRepeatResponseCode);
+    });
+
+    it.skip("BAV CRI /userinfo Happy Path", async () => {
```

Comment:
Can you uncomment the logic we expect to have working when all endpoints have been delivered please? 

For userinfo, we know we'll need to call Stub, session, authorization and token endpoints first.